examples/ligand_in_water/ligand_in_water.ipynb

```diff
@@ -36,8 +36,7 @@
     "from openff.units import unit\n",
```

Comment:
<p>An alternative to having this in an appendix would be to run the whole notebook with two parallel systems. This would emphasize how easy it is to change the force field; instead of seeing a whole appendix with new code, there would just be a few doubled up lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-interchange/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I really like this idea!</p>